src/components/molecules/plan/plan.html

```diff
@@ -0,0 +1,48 @@
+<div class="plan_wrap">
+  <div class="plan">
+    <div class="plan_odd">
```

Comment:
```suggestion
  <ul class="plan">
    <li class="plan_odd">
```
リストの形式になっているパーツの場合、 `ul` `li` を使って書くのがHTML的には望ましいです！
（アクセシビリティやSEOに良い影響を与えます。）